packages/react/src/utils.ts

```diff
@@ -71,3 +71,38 @@ export function useForwardedRef<T extends HTMLElement>(ref: React.ForwardedRef<T
 
   return innerRef;
 }
+
+export class Mutex {
```

Comment:
should we export & use the version from client-sdk-js?

Reply:
makes sense, https://github.com/livekit/client-sdk-js/pull/1110